src/index.js

```diff
@@ -1,4 +1,7 @@
 import React from 'react';
+import './components/TodoComponents/Todo.css';
 import { render } from 'react-dom';
 import App from './App';
```

Comment:
You could just rename your `Todo.css` file to something like "App.css" or "index.css" and move it out of your components folder, since it also handle styles beyond the todo components (like overall app styles).